packages/firebase_messaging/README.md

```diff
@@ -61,18 +61,6 @@ for more.
 
 By default background messaging is not enabled. To handle messages in the background:
 
-1. Add the `com.google.firebase:firebase-messaging` dependency in your app-level `build.gradle` file that is typically located at `<app-name>/android/app/build.gradle`.
```

Comment:
What happens to apps that have already followed this step when they receive this change? Will gradle complain in an actionable way, or is this going to break consumers of the plugin?

Reply:
No there won't be any conflicts, the user would effectively just be redeclaring a dependency already available to them. Gradles dependency resolution will handle everything.